src/nvim/api/ui.c

```diff
@@ -569,3 +644,44 @@ static void remote_ui_inspect(UI *ui, Dictionary *info)
   UIData *data = ui->data;
   PUT(*info, "chan", INTEGER_OBJ((Integer)data->channel_id));
 }
+
+Integer ui_win_move_cursor(Integer direction, Integer count)
+{
+  UI *ui = ext_win_ui;
+  Array args = ARRAY_DICT_INIT;
+  UIData *data;
+  Error error = ERROR_INIT;
+  typval_T rettv;
+
+  if (!ui_is_external(kUIWindows) || ui == NULL) {
+    abort();  // this should never happen
+  }
+  data = ui->data;
+
+  ADD(args, INTEGER_OBJ(direction));
+  ADD(args, INTEGER_OBJ(count));
+
+  Object result = rpc_send_call(data->channel_id, "win_move_cursor", args,
+                                &error);
+
+  if (ERROR_SET(&error)) {
+    api_set_error(&error, kErrorTypeException, "%s", error.msg);
```

Comment:
this just self-assigns the error object, we want something like `EMSG2(_("Error invoking win_move_cursor: %s"), error.msg);`